example/50-seed.yaml

```
@@ -75,6 +75,12 @@ spec:
   # loadBalancerServices:
   #   annotations:
   #     foo: bar
+  #   externalTrafficPolicy: Local
+  #   zoneSettings:
+  #   - zoneName: europe-1a
```

```suggestion
  #   zones:
  #   - name: europe-1a
```

WDYT? Motivation: https://github.com/gardener/gardener/blob/6a0fea86850ffec8937d1956bdf1a8ca6d074f3b/example/30-cloudprofile.yaml#L69-L70

---

Makes sense. I will update it with the next commit.